README.md

```diff
@@ -1,2 +1,33 @@
-# microverse-capstone-1
- HTML/CSS & JavaScript capstone project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Microverse Capstone Project - 1 - AfriCities Summit
+
+ A website showcasing the Africities Summit in Kisumu County, Kenya
+
+## Built With
+
+- Major languages - HTML & CSS
+- Preprocessor - SASS
+
+## Live Demo (if available)
```

Comment:
- [ ] Images in your live page is broken, kindly fix this
![Screenshot from 2022-08-02 22-55-19](https://user-images.githubusercontent.com/66097371/182480397-296af49b-4dd6-4731-aee3-4ae32de20039.png)